### PR TITLE
clarifying description of full per-host images

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
@ares and I were a tad confused by this description and based on the description thought that the provisioning template was embedded into the image.  This isn't the case so I wanted to make this clear.

File: README.md [hunk 1]
```diff
@@ -162,8 +162,9 @@ It must be writable by the 'foreman' user.
 A variant of the per-host image, this version doesn't chainload from Foreman,
 but contains the initial OS bootloader.  It can be useful for hosts that fail
 to chainload correctly, but has the downsides that it may become out of date
-if the host OS, bootloader or templates change, or build tokens are required
-and expire.
+if the host OS, bootloader or PXELinux templates change, or build tokens are
+required and expire.  The provisioning template is still fetched from the
+foreman server.
```
Comment (domcleal) [inline README.md]: `Foreman` should be capitalised

**Comment by domcleal** [comment]
Could you also update the in-app help text please?

**Comment by jlsherrill** [comment]
@domcleal updated in-app help that i found, let me know if there is more i missed

**Comment by domcleal** [comment]
Thanks @jlsherrill, merged.

**Pull request merged**